service/healthcheck/healthcheck.go

```diff
@@ -0,0 +1,39 @@
+package healthcheck
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/ONSdigital/dp-areas-api/models"
+	"github.com/ONSdigital/dp-areas-api/rds"
+	health "github.com/ONSdigital/dp-healthcheck/healthcheck"
+	"github.com/ONSdigital/log.go/v2/log"
+)
+
+var (
+	dbInstanceName = "develop-area-profiles-postgres-one"
```

Comment:
could we make `dbInstanceName` configurable so we could reuse in prod ?

Reply:
👍

Reply:
Do you think we should iterate over all instances and report health or is one instance sufficient?

Reply:
perhaps we can query the cluster endpoint DescribeDBClusters instead of the individual DescribeDBInstances https://github.com/aws/aws-sdk-go/blob/main/service/rds/examples_test.go#L1386-L1411 ?

Reply:
We could do that too - my rationale is `let's attempt to describe a db instance!` - if we can't, we have a problem and want healthcheck state critical

Reply:
checking just one instance sounds good to me !